apps/learner/src/lib/components/FloatingChat.svelte

```diff
@@ -0,0 +1,96 @@
+<script lang="ts">
+  interface Props {
+    /**
+     * The URL to navigate to when the user clicks on the floating chat.
+     */
+    to: string;
```

Comment:
On floating player or chat button clicked, it should open up a full page modal so the props should accept something like `onclick` instead of `to` to navigate.

Floating player side should change the props too.